py_nltk/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM socrata/py_analysis
+
+RUN pip install nltk==3.0.0
+
+RUN mkdir -p /home/socrata
+
+ENV HOME /home/socrata
+
+RUN python -m nltk.downloader -d $HOME/nltk_data stopwords &&\
+    python -m nltk.downloader -d $HOME/nltk_data punkt &&\
+    python -m nltk.downloader -d $HOME/nltk_data wordnet &&\
+    python -m nltk.downloader -d $HOME/nltk_data maxent_treebank_pos_tagger
```

Comment:
From reading http://www.nltk.org/api/nltk.html#module-nltk.downloader, I believe that the best bet would be to host our own internal mirror of the data sets and re-sync with upstream at specified times or based on specified criteria (e.g. we've vetted a new data release for use in our environment and are ready to incorporate it into new builds)

IMO, this is probably not worth doing, but it's up to you and your team to make the final decision based on your assessment of the risk to the service posed by unexpected data changes.